### ISSUE TITLE
Support VRCFaceTracking v5.2.3

### ISSUE THREAD

**Issue by PoptartGhost** [hazre/VRCFTReceiver#5]
Situation : 
It seems the vrcft reciever is empty on the user root.
![image](https://github.com/hazre/VRCFTReceiver/assets/49379196/f361d4eb-1d5e-4468-9f9e-fb62e6ad4507)

I also added a prefab to the avatar but  while its osc reciever is set to 9000 - and vrcft is sending on 9000 - and also i see a 1 m/s message coming IN on vrcft on 9001... I have yet to see any of the paramaters values be driven? or any parameters to load on the vrcftreciever auto added to my root.

Expected behavior:

I think it should be driving the values of each of these 
![image](https://github.com/hazre/VRCFTReceiver/assets/49379196/82ded532-1beb-49f1-9843-5431de007bde)

other things of note: 
- there is a populated .json list in vrc temp /avatars 
- vrcft lists "avatar status as " Loading... but im unsure if thats supposoed to be like that.
- **I am using xr elite , which is using srnipal to communicate to vrcft the same way the vivepro does**
- I cannot use native implementation of SR afaik because XR uses the newest SRanipal.exe 

**ADDITIONAL FINDINGS UPDATE:**
so from talking to people in the resonite discord, the one perosn who know who has it working is on 5.1.1.0 
mine is on 5.3 and there is no way to downgrade.
could this be the issue? all outputs monitored in protokol regaurdless of if we send "force relevancy" or if we send your command of 
![image](https://github.com/hazre/VRCFTReceiver/assets/49379196/d54fdff5-8c77-43aa-b8ef-b928f99f9f96)
"vrc_parameters" 

will send this instead due to the vrc_parameters file not being loaded: 
![image](https://github.com/hazre/VRCFTReceiver/assets/49379196/1dc67474-097d-46ba-8553-b62296967dca)

**Comment by hazre** [comment]
Yes version 5.3 is broken and it can be only accessed using OSCQuery. Please use 5.1.1.0 for now until they fix. There shouldn't be any major changes between the two versions right now.

> so from talking to people in the resonite discord, the one perosn who know who has it working is on 5.1.1.0
mine is on 5.3 and there is no way to downgrade.

You should be able to install 5.1.1 side by side with 5.3.0, that's what I do currently.

**Comment by hazre** [comment]
I've updated the readme to reflect that.

**Comment by hazre** [comment]
Figured it out how to run/install 5.1.1 even though the certificate has expired. Download the MSIX file, rename the extension to .zip and export it in a folder. Then you should be able to run VRCFaceTracking.exe in that folder to launch VRCFT v5.1.1

**Comment by PoptartGhost** [comment]
omg wow okay ill try that thank you so much!

**Comment by hazre** [comment]
I've also fixed up the issue in a PR, hopefully it gets merged soon


https://github.com/benaclejames/VRCFaceTracking/pull/215

**Comment by AwesomeTornado** [comment]
It looks like your PR got merged, has this fixed the issue? I am trying to get PSVR2 eye tracking working on Resonite, but the PSVR2 module only works on VRCFT V5.2.3.0, not on V5.1.1.0

**Comment by AwesomeTornado** [comment]
It seems like the avatar status is perpetually stuck on "loading parameters"

Ok, so your PR hasn't been included in the latest release, but after building from source it seems to recognize the avatar parameters. I was unable to get those parameters to show up in Resonite, but I will try more tonight.

**Comment by AwesomeTornado** [comment]
Ok, it seems that OSC message batching is causing issues with this mod. VRCFT is sending multiple OSC messages in a single packet, and Resonite VRCFT is reading the #bundle at the beginning of the packet, and throwing it out. By changing VRCFT to send one packet per update, Resonite VRCFT now seems to work for me.

**Comment by AwesomeTornado** [comment]
You can try out my custom version here
https://github.com/AwesomeTornado/VRCFaceTracking
I might make a pr to get this added as a default toggle, especially because its mentioned in the code that this would be a possible feature in the future.

**Comment by hazre** [comment]
Hmm interesting I would need look if it's possible to fix that in the mod instead. I might just go back to the v1 way of processing osc messages
